### ISSUE TITLE
[Keyboard Navigation – BotKit]: Underline is not defined for the header links in the page.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/botkit-website#15]
**User Experience:**
Users who rely on keyboards will affect to navigate, if underline is not defined for the links when focus is on links.

**Test Environment:**
Url: https://botkit.ai/
OS: Version 1809(OS Build 17763.379)
Browser: Edge
Ease of Access: Settings->High contrast modes

**Repro Steps:** 
Step 1: Open the above URL in Edge browser.
Step 2: Navigate to “Get Started, Docs, Support and Blog” links using keyboard tab key.
Step 3: Observe whether underline is defined for “Get Started, Docs, Support and Blog” links.

**Actual Result:**
Underline is not coming when user navigating “Get Started, Docs, Support and Blog” links using keyboard tab key.

**Expected Result:**
Underline should be visible when user navigating “Get Started, Docs, Support and Blog” links using keyboard tab key.
Ex: When keyboard focus is on "Get Started" link, underline should be visible.

**Note**: Chevron symbol should be defined for "Get Started" and "View on GitHub" links in "BotKit"  page.
When Narrator focus is on "Get Started" link, Narrator should narrate "Get Started open in new tab link".
Please refer the attachment as <MAS1.4.1_BotKit_Chevron.png>

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={9f1f0271-42a1-45b1-be00-31473ba2e579}](url)

[MAS1.4.1_BotKit.zip](https://github.com/howdyai/botkit-website/files/2977522/MAS1.4.1_BotKit.zip)

![MAS1 4 1_BotKit_Chevron](https://user-images.githubusercontent.com/48272005/54528541-4fdea900-49a3-11e9-9496-5a9b1222967e.png)

**Comment by benbrown** [comment]
@awalia13 I'm not sure what this comment means:

"Chevron symbol should be defined for "Get Started" and "View on GitHub" links in "BotKit" page."

**Comment by benbrown** [comment]
Links in header will now be underlined when focused.

**Comment by v-gasikk** [comment]
Issue is partially Fixed. 
Now Underline is defined for the header links in the page - issue is Fixed
Chevron symbol should be defined for "Get Started" and "View on GitHub" links in "BotKit" page.  - - Not Fixed.
Because it's looking like as button, so there should be identification like chevron symbol (>) to identify them as link

**Comment by benbrown** [comment]
Chevron symbol has been added to buttons on homepage.

**Comment by v-gasikk** [comment]
Verified. Issue is Fixed. Hence Closing the bug.
